include/pipewire_screencast.h

```diff
@@ -3,6 +3,11 @@
 
 #include "screencast_common.h"
 
+#ifdef __FreeBSD__
+/* sys/param.h also defines ALIGN */
+#undef ALIGN
+#endif
+
 #define BUFFERS 1
 #define ALIGN 16
```

Comment:
Hm, can we rename our macro instead?

Reply:
Renaming helps. I've used the same prefix as functions in pipewire_screencast.h.